### PR TITLE
added details for regestration

### PR DESCRIPTION
**Pull request by ravisaxena23**
you are great please review my PR and suggest me changes else please accept my PR.

### CODE REVIEW

File: content/participants/ttjc@ravisaxena23/index.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+type: "participant"
+name: "Ravi Nandan Saxena"
+portfolio: "http://bit.ly/ravisresume"
+github-link: "https://github.com/ravisaxena23"
+linkedin-link: "https://www.linkedin.com/in/ravisaxena23/"
+looking-for: "job"
+description: "expert at CSS/HTML intermediate in React/Node learning flutter/ReactNative"
+---
+
+Hi! I am a Ravi Nandan Saxena a Web developer let me walk you through my intro, projects and blogs.
```
Comment (khusharth) [inline content/participants/ttjc@ravisaxena23/index.md:12]: Hey! @ravisaxena23 a small mistake here:
* I am **a** Ravi Nandan

File: content/participants/ttjc@ravisaxena23/index.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+
+If I don't know something I love to learn it from anyone anywhere. I have watched almost all sci-fi seasons present on netflix lol.
+
+### Help I need?
+
+It would be good if someone tells me how to do auth for apps.
+
+### How to contact me?
+
+- Twitter
```
Comment (khusharth) [inline content/participants/ttjc@ravisaxena23/index.md:33]: Please also add the respective links to all the mentioned platforms using the markdown syntax.

File: content/participants/ttjc@ravisaxena23/index.md [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+- LinkedIn
+- E-mail
+- Call
+
+### Projects
+
+not a lot of projects are done. I will keep updating it with real links when done.
+
+My projects: _github link_ https://github.com/ravisaxena23/technojam-frontend
```
Comment (khusharth) [inline content/participants/ttjc@ravisaxena23/index.md:42]: * Please follow the registration video format to add details about your project. 
* Also only projects done during the challenge are accepted. Previous projects are not accepted so please remove them.

File: content/participants/ttjc@ravisaxena23/index.md [hunk 4]
```diff
@@ -0,0 +1,70 @@
+---
+title: Ravi Nandan Saxena
+type: "participant"
+name: "Ravi Nandan Saxena"
+portfolio: "http://bit.ly/ravisresume"
```
Comment (khusharth) [inline content/participants/ttjc@ravisaxena23/index.md:5]: Portfolio should contain the link to your own personal website not your resume. If you dont have one right now leave it blank and update it later.

File: content/participants/ttjc@ravisaxena23/index.md [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+
+### Projects
+
+not a lot of projects are done. I will keep updating it with real links when done.
+
+My projects: _github link_ https://github.com/ravisaxena23/technojam-frontend
+             _github link_ https://github.com/ravisaxena23/crypto-ciphers
+
+
+#### Binder: tinder for books
```
Comment (khusharth) [inline content/participants/ttjc@ravisaxena23/index.md:46]: Also remove this project and the blog below. These are just meant to show you the format. You need to add your own projects and blog here.

**Comment by ravisaxena23** [comment]
hey @khusharth thank you for reviewing please check again and give suggestion

File: content/participants/ttjc@ravisaxena23/index.md [hunk 6]
```diff
@@ -30,18 +30,14 @@ It would be good if someone tells me how to do auth for apps.
 
 ### How to contact me?
 
-- Twitter
-- LinkedIn
-- E-mail
-- Call
+- LinkedIn https://www.linkedin.com/in/ravisaxena23/
```
Comment (khusharth) [inline content/participants/ttjc@ravisaxena23/index.md:33]: Use the markdown format to add all the links. Example:
```
[LinkedIn](https://www.linkedin.com/in/ravisaxena23/)
```

File: content/participants/ttjc@ravisaxena23/index.md [hunk 7]
```diff
@@ -30,18 +30,14 @@ It would be good if someone tells me how to do auth for apps.
...
 
 ### Projects
 
 not a lot of projects are done. I will keep updating it with real links when done.
 
-My projects: _github link_ https://github.com/ravisaxena23/technojam-frontend
-             _github link_ https://github.com/ravisaxena23/crypto-ciphers
-
 
 #### Binder: tinder for books
```
Comment (khusharth) [inline content/participants/ttjc@ravisaxena23/index.md:42]: Remove this project (Binder) and the blog below it. These are just meant to show you the format. You need to add your own projects and blog here.

Comment (ravisaxena23) [reply content/participants/ttjc@ravisaxena23/index.md]: should i add this thing in the base readme? it may help others although i have not added it now

**Comment by ravisaxena23** [comment]
i have done changes please re review.. thank you

**Comment by ravisaxena23** [comment]
@khusharth hey is it good to be accepted?

File: content/participants/ttjc@ravisaxena23/index.md (hunk 7 above)
Comment (khusharth) [reply content/participants/ttjc@ravisaxena23/index.md]: Its clearly mentioned in the registration video to keep this place empty.

**Comment by khusharth** [comment]
@ravisaxena23 Looks fine now.
@gkanishk Good to merge.

**Pull request merged**